src/manifest.json

```diff
@@ -1,6 +1,7 @@
 {
   "manifest_version": 3,
   "name": "Save to Pocket",
+  "default_locale": "en",
```

Comment:
required to have default locale per the documentation:
https://developer.chrome.com/docs/extensions/reference/i18n/#how-to-support-multiple-languages